prql-java/cross.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+PRQL_JAVA_MODULE=$1
+echo PRQL_JAVA_MODULE="${PRQL_JAVA_MODULE}"
+CONTEXT_PATH=$(pwd)
+echo CONTEXT_PATH="${CONTEXT_PATH}"
+cd "${PRQL_JAVA_MODULE}" || exit 1
+
+# install cross
+cargo install cross
+
+# x86_64-unknown-linux-gnu
+echo "compiling for x86_64-unknown-linux-gnu"
```

Comment:
My guess is that this file could move to a workflow eventually; but to the extent you've been iterating on it locally, it looks good, and we can move to a workflow later.